tests/linter_tests/test_usfm_linter.py

```diff
@@ -71,7 +71,7 @@ def test_PhpDuplicateUsfmFileName(self):
         shutil.copy(os.path.join(out_dir, self.php_file_name), os.path.join(out_dir, 'PHP.usfm'))
         expected_warnings = True
         linter = self.run_linter(out_dir)
-        self.assertEqual(linter.log.warnings[0], u"File './PHP.usfm' has same code 'PHP' as previous file")
+        self.assertEqual(linter.log.warnings[0], "File '."+os.path.sep+"PHP.usfm' has same code 'PHP' as previous file")
```

Comment:
OK, but may not be necessary to remove /, since it will also work on windows.

Reply:
It didn't work on my Windows...it put .\PHP.usfm as the path.